packages/ndla-ui/src/locale/messages-nn.js

```diff
@@ -660,6 +660,21 @@ const messages = {
       linkText: 'Gå til ndla.no for dagens innhald',
     },
   },
+  frontPageToolbox: {
+    heading: 'Verktøykassa',
+    text:
+      'Har du lyst til å bli god til å presentera, eller vil du læra å studera smartare ved hjelp av riktig studieteknikk? Treng du råd om korleis du les mest mogleg effektivt til eksamen? I Verktøyskassen til NDLA finn du masse gode tips og råd!',
```

Comment:
```suggestion
      'Har du lyst til å bli god til å presentere, eller vil du lære å studere smartare ved hjelp av riktig studieteknikk? Treng du råd om korleis du les mest mogleg effektivt til eksamen? I Verktøyskassa til NDLA finn du masse gode tips og råd!',
```
Usikker på om det skal være liten eller stor V i siste setninga.

Reply:
liten v i verktøykassa. Men verktøykassa, ikkje verktøyskassa/verktøyskassen (ein ekstra s)

Reply:
Endringer committa